src/main/java/org/codewars/utils/TaskRunner.java

```diff
@@ -3,17 +3,26 @@
 import java.math.BigInteger;
 
 public class TaskRunner {
+
+    private final Reader reader = new Reader();
+    private final Users user;
     public static void setUser(int id){
         // TODO: Implement setUser() method
     }
     public static long[] runGap() {
         return new long[0];
     }
 
-    public static BigInteger runPerimeter(){
-        return new BigInteger("0");
-    }
-
+//    public static BigInteger runPerimeter(){
+//        return new BigInteger("0");
+//    }
+public static void runPerimeter() {
+    System.out.println("Run task perimeter");
+    System.out.println("Input the number of squares in the rectangle ");
+    BigInteger x = reader.readBigInteger();
+    BigInteger result = user.getFiveImpl().perimeter(x);
+    System.out.println(result);
```

Comment:
mb add some text here to show that this is result (sum of perimeters of such squares)?

Reply:
do you mean sm like this?
 System.out.println("Sum of the perimeters of all the squares" + result);

Reply:
yes

Reply:
done